graphqlmetrics/config/config.go

```diff
@@ -35,6 +35,10 @@ type Config struct {
 	IsPrometheusEnabled  bool   `default:"false" envconfig:"PROMETHEUS_ENABLED"`
 	PrometheusListenAddr string `default:"127.0.0.1:8088" envconfig:"PROMETHEUS_LISTEN_ADDR"`
 	PrometheusPath       string `default:"/metrics" envconfig:"PROMETHEUS_PATH"`
+
+	MaxCostThreshold        int           `default:"10000" envconfig:"MAX_COST_THRESHOLD"`
```

Comment:
Please prefix the environment variables to understand that they belong together.